ComboIndicator/ComboIndicator.cs

```diff
@@ -58,10 +60,10 @@ public class ComboIndicator : MonoBehaviour {
 		}
 
 		void LateUpdate() {
-			config = Config.LoadConfig();
```

Comment:
Not sure if there was a reason the config was getting loaded here as well but this was one of the hardest hits.